newIDE/app/src/MainFrame/EditorContainers/EventsEditorContainer.js

```diff
@@ -62,25 +63,27 @@ export class EventsEditorContainer extends React.Component<RenderEditorContainer
     }
 
     return (
-      <EventsSheet
-        ref={editor => (this.editor = editor)}
-        setToolbar={this.props.setToolbar}
-        onOpenLayout={this.props.onOpenLayout}
-        resourceSources={this.props.resourceSources}
-        onChooseResource={this.props.onChooseResource}
-        resourceExternalEditors={this.props.resourceExternalEditors}
-        openInstructionOrExpression={this.props.openInstructionOrExpression}
-        onCreateEventsFunction={this.props.onCreateEventsFunction}
-        unsavedChanges={this.props.unsavedChanges}
-        project={project}
-        scope={{
-          layout,
-        }}
-        globalObjectsContainer={project}
-        objectsContainer={layout}
-        events={layout.getEvents()}
-        onOpenExternalEvents={this.props.onOpenExternalEvents}
-      />
+      <LeaderboardProvider gameId={project.getProjectUuid()}>
```

Comment:
As discussed yesterday together, is it here the best place to put the provider? If you use the action in an external events editor, will it work?